qemu/tests/block_copy.py

```diff
@@ -314,12 +314,15 @@ def wait_for_finished(self):
         """
         waiting until block job finished
         """
+        time_start = time.time()
         params = self.parser_test_args()
         timeout = params.get("wait_timeout")
         finished = utils_misc.wait_for(self.job_finished, timeout=timeout)
         if not finished:
             raise error.TestFail("Job not finished in %s seconds" % timeout)
+        time_end = time.time()
```

Comment:
the code you added will affect all the test cases using wait_for_finished.
although it works to get execution time, but i suppose it will affect other cases using this function.
what do you think to get time in your specific case?
like:
get time
wait_for_finished
get time